Formula/xdg-ninja.rb

```diff
@@ -0,0 +1,21 @@
+class XdgNinja < Formula
+  desc "Check your $HOME for unwanted files and directories"
+  homepage "https://github.com/b3nj5m1n/xdg-ninja"
+  url "https://github.com/b3nj5m1n/xdg-ninja/archive/refs/tags/v0.2.0.1.tar.gz"
+  sha256 "f4f9ab4500e7cf865ff8b68c343537e27b9ff1e6068cb1387e516e608f77cec8"
+  license "MIT"
+  head "https://github.com/b3nj5m1n/xdg-ninja.git", branch: "main"
+
+  depends_on "glow"
+  depends_on "jq"
+
+  def install
+    prefix.install "programs/"
```

Comment:
I wonder if it makes more sense to install this into `pkgshare`.

Reply:
I didn't really know where to put everything. The important part is that `"programs/" and `"xdg-ninja.sh"` must be installed in the same directory.

I can change `prefix` to `pkgshare` if that would be better.

Reply:
Yea, `pkgshare` seems better.

Reply:
Cool. I've pushed a commit with those changes.